app/Resources/views/archive-month.html.twig

```diff
@@ -6,13 +6,15 @@
 
     {{ render_pattern(contentHeader) }}
 
-    {% if covers %}
+    <div class="wrapper">
```

Comment:
Shouldn't be need when there's no `covers`, but there's no space at all if it's not there. Should be changed, but ignoring for now.